.travis.yml

```diff
@@ -3,6 +3,7 @@ python: 2.7
 env:
   - TOX_ENV=py27-dj15
   - TOX_ENV=py27-dj16
+  - TOX_ENV=py27-dj17
   - TOX_ENV=pep8
```

Comment:
could you add the tox env to run on py37 / dj2.2?

Reply:
added